int32 id
---
float32[6] setpos